fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/AbstractResource.java

```diff
@@ -58,8 +58,7 @@ public class AbstractResource {
     @Optional
     protected Supplier<String> pidMinter;
 
-    // Mint non-hierarchical identifiers. To force pairtree creation as default, use
-    //  ConfigurableHierarchicalSupplier(int length, count) instead.
+    @Inject
     protected UniqueValueSupplier defaultPidMinter = new ConfigurableHierarchicalSupplier();
```

Comment:
Don't need `@Inject` here, you are instantiating a `new ConfigurableHierarchicalSupplier();` anyways.